CHANGELOG.md

```diff
@@ -3,12 +3,13 @@
 
 ## Unreleased
 
-### `Changed`
+### Added ability to select and copy text in a tree
 
-- Tree layout selection is now a drop down menu and not a slider. [PR 2](https://github.com/phac-nml/ArborView/pull/2)
+- The SVG text is not selectable and could be copied. This makes it easy for researchers to highlight elements of interest such as accession number, etc. This is both valid for interactive mode and even during SVG image exports [PR 8](https://github.com/phac-nml/ArborView/pull/8)
+- The text selection cursor symbol changes from hand (before) to a pipe symbol (`|`) in both the webapp GUI and SVG image export. Injected CSS style in svg tag which is much safer on line 453: `svg.append("defs").append("style").attr("type", "text/css").text("svg text {cursor: text}>"); `
```

Comment:
Thank you so much for the verbose changelog messages, but can you remove the second bullet point and perhaps summarize it in the first one if needed? 

As the line numbers will change as we add more code, and the link to the PR will allow us to see what code was changed in the diff.

Reply:
fixed in: https://github.com/phac-nml/ArborView/pull/8/commits/2b3fd3773a82a0498870253e52f853fcc7e0a134